src/cuttlefish_escript.erl

```diff
@@ -230,6 +222,47 @@ describe(ParsedArgs, [Query|_]) when is_list(Query) ->
     end,
     stop_deactivate().
 
+%% This is the function that dumps the value for a single setting.
+%% To be shell friendly, it returns nothing when something is wrong.
+get(_ParsedArgs, []) ->
+    %% No query, you get nothing.
+    ?logger:debug("cuttlefish's get command required a variable to query.", []),
+    ?logger:debug("Try `describe setting.name`", []),
+    stop_deactivate();
+get(ParsedArgs, [Query|_]) when is_list(Query) ->
+    ?logger:debug("cuttlefish get '~s'", [Query]),
+    case match(ParsedArgs, Query) of
+        {error, notfound} ->
+            ?logger:debug("Variable '~s' not found", [Query]);
+        {ok, Match} ->
+            Conf = load_conf(ParsedArgs),
+            case lists:keyfind(cuttlefish_variable:tokenize(Query), 1, Conf) of
+                false ->
+                    ConfFile = proplists:get_value(conf_file, ParsedArgs),
+                    ?logger:debug("Value not set in ~s", [ConfFile]);
+                {_, CValue} ->
+                    ConfiguredValue = format_datatype(CValue, cuttlefish_mapping:datatype(Match)),
+                    ?STDOUT("~s", [ConfiguredValue])
+            end
+    end,
+    stop_deactivate().
```

Comment:
this will cause escript to exit with code `1`.
maybe call `stop_ok()` at line 245 and `stop_deactivate()` in other clauses.

Reply:
init:stop is kind of slow, even worse, stop_deactivate/0 has a sleep for 250ms. 
I think we can change to call `halt(0)` or `halt(1)` instead

Reply:
fixed